server/src/builder.rs

```diff
@@ -42,16 +44,19 @@ type EdgeInfo = (
 fn build_caches() -> CacheContainer {
     let token_cache: DashMap<String, EdgeToken> = DashMap::default();
     let features_cache: DashMap<String, ClientFeatures> = DashMap::default();
+    let delta_cache: DashMap<String, DeltaCache> = DashMap::default();
     let engine_cache: DashMap<String, EngineState> = DashMap::default();
     (
         Arc::new(token_cache),
         Arc::new(FeatureCache::new(features_cache)),
+        Arc::new(delta_cache),
         Arc::new(engine_cache),
     )
 }
 
 async fn hydrate_from_persistent_storage(cache: CacheContainer, storage: Arc<dyn EdgePersistence>) {
-    let (token_cache, features_cache, engine_cache) = cache;
+    let (token_cache, features_cache, _delta_cache, engine_cache) = cache;
+    // TODO: do we need to hydrate from persistant storage for delta?
```

Comment:
Yes I think so. Otherwise when pods spin up they'll send empty responses until they hydrate. Not sure how we intend to handle that yet but I guess that's next PR